### DISCUSSION TITLE
Max brightness button or brightness slider for photos viewer

### DISCUSSION THREAD

**Discussion by JKP2599** [deckerst/aves#1186] [Ideas]
It would be good if we were able to turn on max brightness briefly for a photo or turn up the brightness slightly without having it maxed all the time.
Hdr photos look better at higher brightness I guess this would help

**Comment by deckerst** [comment]
Related to this, you can select `Settings` > `Display` > `Maximum Brightness` > `Viewer page only`. It's a general setting, not dynamic with a gesture, but maybe it will be useful to you. 

Also, Aves is currently unable to properly display HDR photos and videos with their full range, so they will likely be under/over exposed.
